docs/userguides/profile.md

```diff
@@ -15,6 +17,19 @@ Your password is not shown when you do `code42 profile show`. However, `code42 p
 password exists for your profile. If you do not set a password, you will be securely prompted to enter a password each
 time you run a command.
 
+## API client authentication
+
+Once you've generated an API Client in your Code42 console, use the following command to create your profile with API client authentication:
+```bash
+code42 profile create-api-client --name MY_API_CLIENT_PROFILE --server example.authority.com --api-client-id "key-42" --secret "code42%api%client%secret"
+```
+
+```{eval-rst}
+.. note:: Remember to escape special characters in your API client secret like `!` with a backslash. ex: `"my!secret"` should be passed in as `"my\!secret"`
```

Comment:
let's just suggest always wrapping the secret in single-quotes to prevent any shell expansion issues

Reply:
I did not know this about single vs double quotes in bash.. thats a good tip